SPECS/wget/wget.spec

```diff
@@ -9,8 +9,23 @@ Vendor:         Microsoft Corporation
 Distribution:   Mariner
 Source0:        https://ftp.gnu.org/gnu/%{name}/%{name}-%{version}.tar.gz
 BuildRequires:  openssl-devel
+BuildRequires:  perl(JSON::PP)
+BuildRequires:  perl(Module::Runtime)
+BuildRequires:  perl(Pod::Coverage)
+BuildRequires:  perl(Sub::Name)
+BuildRequires:  perl(YAML)
+BuildRequires:  perl(autodie)
```

Comment:
If these are `%check` dependencies, we should gate them under the `with_check` conditional.
If they are not, does this mean, our `wget` never built with the normal build ?